job_spec/WATER_MAP.yml

```diff
@@ -83,9 +83,10 @@ WATER_MAP:
         type: number
     known_water_threshold:
       api_schema:
-        description: Threshold for extracting known water area in percent. Ignored when flood_depth_estimator is None.
-        default: 30.0
+        description: Threshold for extracting known water area in percent. A water threshold is computed when value is null. Ignored when flood_depth_estimator is None.
+        default: null
         type: number
+        nullable: true
```

Comment:
Is this worth including in the Changelog entry?

Reply:
Oh yes, yes it is.